packages/form-footer/src/FormFooter.tsx

```diff
@@ -143,6 +143,7 @@ export default function FormFooter({
   const { theme, darkMode } = useDarkMode(darkModeProp);
   return (
     <footer
+      data-testid="lg-form-footer-footer"
```

Comment:
nit: `lg-form_footer-footer` (use underscores to separate words in the same block, dashes to separate blocks)